Documenting workspace's disks and updated the template file (#2566)
alan-turing-institute/data-safe-haven: docs/source/roles/researcher/using_the_sre.md

> (role_researcher_shared_storage)=

Reviewer: Let's check where this is being used. Does shared here imply shared across all workspaces?

Author: Yeah, I think that's the implication.  For example: `For more information on shared SRE storage volumes, consult the {ref}`Safe Haven User Guide <role_researcher_shared_storage>`.`

Reviewer: In that case, I think this label belongs with the section on sharing files. It is currently linking to the wrong place.

Author: I think it is linking to the "Sharing Files" section. Should it be pointing somewhere else? 
<img width="937" height="193" alt="image" src="https://github.com/user-attachments/assets/3e1bdbae-42da-4ecb-93b2-bcbd412f80ce" />

> ## {{computer_disk}} Working with disks

Reviewer: ```suggestion
## {{computer_disk}} High performance storage
```

I don't think the important thing for the researcher is that this is an Azure disk, but more that this is the fast storage, and a warning that this storage is local to each workspace.

> The following disks can be available to a workspace within the SRE:

Reviewer: ```suggestion
Fast storage may be available to a workspace within the SRE.
This storage has good performance for work reading, writing or manipulating files.
However, these directories are **local to each workspace** so any files will not be visible on another workspace.
```

> - The *operating system disk*, available at `/`, contains the operating system. As a researcher, **you are not able to write to this disk**. If by any chance you require access, for example you might need to install software as root, please get in touch with your System Manager.

Reviewer: I'm not sure if this is correct. `/var` and `/tmp` may be on that disk (or a tmpfs) which normal user can write to (though maybe indirectly).

That said, I don't think we need to explain it because we don't really expect users to interact with it much.

Author: Just checked, and you can write to `/tmp` as a researcher. I'm going to rephrase this

> - The *data disk*, at `/mnt/datadrive`, is available to all researchers and it is meant to store data with good read/write performance. If present (if not, talk to your System Manager) each data disk **is attached to one workstation only** and can only be accessed from it. If you want to share information between workspaces, please check the "Sharing files inside the SRE" section below.

Reviewer: ```suggestion
- The *data disk*, at `/mnt/datadrive`, is available to all researchers and it is meant to store data with good read/write performance.
  If present (if not, talk to your System Manager) each data disk **is attached to one workstation only** and can only be accessed from it.
  If you want to share information between workspaces, please check the "Sharing files inside the SRE" section below.
```

Let's add a cross reference for System Manager.

Reviewer: And the sharing files section.

> - The *temporary disk* is not available to every workspace. When present, researchers can write and read to `/mnt/scratch`. Write and read performance to the temporary disk is very good, however, it is only meant for short-term storage. Data can be lost during a maintenance event, like a redeploy. As with a data disk, temporary disks are attached to a specific workspace and can only be accessed from it.

Reviewer: ```suggestion
- The *temporary disk* is not available to every workspace.
  When present, researchers can write and read to `/mnt/scratch`.
  Write and read performance to the temporary disk is very good, however, it is only meant for short-term storage.
  Data can be lost during a maintenance event, like a redeploy.
  As with a data disk, temporary disks are attached to a specific workspace and can only be accessed from it.
```

(role_researcher_shared_storage)=

(role_researcher_high_performance_storage)=

## {{computer_disk}} High performance storage

Fast storage may be available to a workspace within the SRE.
This storage has good performance for work reading, writing or manipulating files.
However, these directories are **local to each workspace** so any files will not be visible on another workspace.

> - The *operating system disk*, available at `/`, contains the operating system. As a researcher, **you have very limited write access to this disk**. If by any chance you require access, for example you might need to install software as root, please get in touch with your {ref}`system manager <role_system_manager>`.

Reviewer: ```suggestion
```

I would say delete this. It isn't storage we are suggesting researchers should use.

Author: Yeah... makes sense.
